.github/workflows/create_release.yaml

```diff
@@ -12,7 +12,7 @@ on:
         type: string
         description: The version of the release
       body:
-        required: true
+        required: false
```

Comment:
let's leave this as true. it's a best practice to always have a release body so we can enforce that with types

Reply:
should we have a release body for nightly-release as well?

Reply:
Added release body for nightly build and reverted the body param to true.